docs/cli/README.md

```diff
@@ -12,10 +12,25 @@ nio new [your project name]
 
 ---
 
-## [Add blocks to your project:](add.md)
+## [Add blocks to your project](add.md)
 
 ```bash
 nio add [block name]
 ```
 
+---
+
+## [List services in a running instance](list.md)
+
+```bash
+nio ls services
+```
+
+---
+
+## [Start/Stop services in a running instance](command.md)
+
+```bash
+nio command start <service-id>
```

Comment:
What about other commands?

Reply:
I think for an overview, one basic command is alright, but I would like to add more examples to the docs/cli/commands.md file